components/resources/quizzes/hirabilityQuiz/EngineerSelection.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from "react";
```

Comment:
This component is built off of SkillSelection.tsx, but is only responsible for the engineering selection to avoid confusing the two.